ThirdPartyAdapters/vungle/vungle/build.gradle

```diff
@@ -22,8 +22,8 @@ android {
     }
     defaultConfig {
         minSdkVersion 16
-        targetSdkVersion 28
-        versionCode 6070000
+        targetSdkVersion 29
```

Comment:
@jiancchen we are making our SDK targetSdkVersion set to 30 , I do realize that 30 is not out yet and we might be safer one down in adapters , do you see any issue if we set this as 30, afaik targetSdkVersion set by Pub at App level should override this anyways